feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.ts

```diff
@@ -61,9 +62,19 @@ export class ConfiguratorOverviewMenuComponent implements AfterViewInit {
   }
 
   protected getHeight(): string {
```

Comment:
Please also document the meaning of that method

Reply:
Done!